src/main/webapp/app/entities/event/event.tsx

```diff
@@ -57,143 +69,64 @@ export class Event extends React.Component<IEventProps, IEventState> {
     this.props.getEntities(activePage - 1, itemsPerPage, `${sort},${order}`);
   };
 
+  toggleModal = eventId => {
```

Comment:
method name toggle but you are doing open modal

Reply:
Resolve in 7f7222ecea72e3c604f27767af7c3e4f433c1c1d